reference/factorization/par_ilut_kernels.cpp

```diff
@@ -191,7 +191,12 @@ void threshold_filter_approx(std::shared_ptr<const DefaultExecutor> exec,
     // pick splitters
     for (IndexType i = 0; i < bucket_count - 1; ++i) {
         // shift by one so we get upper bounds for the buckets
-        sample[i] = sample[(i + 1) * sampleselect_oversampling];
+        // TODO FIXME: NVHPC 23.3 seems to handle assignment index with
```

Comment:
wow, this is pretty bad... BTW it is fixed in NVHPC 24.1, maybe also mention that.